src/Umbraco.Web.UI/umbraco/config/lang/en.xml

```diff
@@ -170,6 +170,7 @@
     <key alias="updateDate">Last edited</key>
     <key alias="updateDateDesc" version="7.0">Date/time this document was edited</key>
     <key alias="uploadClear">Remove file(s)</key>
+    <key alias="url">URL</key>

```

Comment:
@abryukhov I think the time will be better spent, doing it for v8. There will be a lot of changes by then - especially since that release is mostly about removing old stuff and cleaning up stuff that will no longer be used.
I'm hoping there will also be some time to do a major cleanup of the localization files when we do that! :)